src/foam/dao/history/HistoryRecord.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.history',
+  name: 'HistoryRecord',
+  documentation: `Contains an array of property updates`,
+  properties: [
```

Comment:
You also need to store the 'id' of the object which was updated. Maybe call it 'objectId' or 'key'.